src/lib/theme/plugin.ts

```diff
@@ -54,17 +63,20 @@ const dark: Partial<Variables<keyof typeof variables>> = {
 
   '--color-surface-primary': rgb(colors.black),
   '--color-surface-secondary': rgb(colors.offBlack),
+  '--color-surface-interactive-secondary': rgb(getColor('rhino', 950)),
   '--color-surface-subtle': rgb(getColor('slate', 900)),
   '--color-surface-badge': rgb(getColor('slate', 700)),
   '--color-surface-information': rgb(getColor('blue', 950)),
-  '--color-surface-warning': rgb(getColor('yellow', 950)),
+  '--color-surface-warning': rgb(getColor('red', 300)),
```

Comment:
I wonder if we should keep `warning` as is, and add a `danger` or `destructive` var for red-300. Maybe @stevekinney has thoughts?

Reply:
I agree.